### ISSUE TITLE
DB & MQ retry logic

### ISSUE THREAD

**Issue by silverdaz** [NBISweden/LocalEGA#381]
Improve the robustness of the DB and MQ connections

**Comment by juhtornr** [comment]
@silverdaz Makes sense but can you be little bit more specific? What is the problem at the moment?
